instagram-app/src/App.js

```diff
@@ -10,21 +10,65 @@ import "./App.css";
 //data
 import dummyData from "./dummy-data";
 
-function App() {
-  return (
-    <div className="App">
-      <header>
-        <SearchBar />
-      </header>
-      <div>
-        <h1>
-          {dummyData.map(data => (
-            <PostContainer key={data.id} dataOnProps={data} />
-          ))}
-        </h1>
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      userData: [],
+      username: "",
+      display: ""
+    };
+  }
+
+  componentDidMount() {
+    dummyData.map(user => {
+      return (user.display = "");
+    });
+    this.setState({
+      userData: dummyData,
+      display: ""
+    });
+  }
+
+  search = e => {
+    const searchTerm = e.target.value;
+
+    this.state.userData.map(user => {
```

Comment:
this is what I originally had but I couldn't get it to work... i was returning user.display = "no" instead of setting that and then returning user. duh! thank you!